sonaranalyzer-dotnet/src/SonarAnalyzer.CFG/ControlFlowGraph/Blocks/Block.cs

```diff
@@ -53,7 +53,7 @@ protected Block()
 
         public virtual IReadOnlyList<Block> SuccessorBlocks { get; } = ImmutableArray.Create<Block>();
 
-        internal IList<SyntaxNode> ReversedInstructions { get; } = new List<SyntaxNode>();
+        public IList<SyntaxNode> ReversedInstructions { get; } = new List<SyntaxNode>();
```

Comment:
We could add a FIXME here to put back the original `internal` accessibility level once we have moved the unit tests to this project too